x-pack/plugins/transform/public/app/sections/transform_management/components/transform_list/use_actions.tsx

```diff
@@ -42,6 +43,8 @@ export const useActions = ({
     modals: (
       <>
         {startAction.isModalVisible && <StartActionModal {...startAction} />}
+        {stopAction.isModalVisible && <StopActionModal {...stopAction} />}
+
         {editAction.config && editAction.isFlyoutVisible && (
```

Comment:
We should also add the warning if the user tries to edit the transform.

Reply:
Updated here https://github.com/elastic/kibana/pull/122305/commits/a7f844ba5fe22eea22f9798812b7e2b0d5aa5e11